frontend/src/css/Login.css

```diff
@@ -0,0 +1,19 @@
+.login {
+  width: 50%;
+  height: 40vh;
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  align-items: center;
+  justify-content: space-evenly;
+  margin-bottom: 15vh;
+}
+
+.modal {
+  position: fixed !important;
```

Comment:
theres probably a better way to do this

Reply:
We are going change the way we are going to be doing CSS so this is a temp style